### ISSUE TITLE
Windows machine upgrades/optimizations/software

### ISSUE THREAD

**Issue by ramiabraham** [norcross/ama#2]
I recently built a PC, but for the past decade or so, I've been a Mac and Linux user almost exclusively.

I'm very happy with the hardware side of things - but am now on the mission of tracking down software, and am having trouble deciding on or finding this stuff:

[System build info, PCPartPicker.com](http://pcpartpicker.com/p/W4FhVn)

Long-ass list of questions:
- What's a solid anti-virus? Do I even need one if I'm totes super careful these days? Is ESET still la good choice?
- For some reason I can't recall the syntax editor/IDE you use. What's your syntax editor of choice? I'm just using ST3, but wondering if there's something better for Windows. The last time I wrote code to any large degree on Windows, I was using Visual Studio heh
- What do you use for command line?
- Have you had any issues with Vagrant/VVV setup on Windows?
- Is there a good performance monitor/optimizer program for Windows? Features like a mem defragger, file shredder, etc.
- Any general apps or tips for a solid dev environment you recommend?

gracias

**Comment by norcross** [comment]
I've been on Mac now for about a year or so, but still have some Windows stuff around the house (and was Windows-only for many years prior). Here's the rundown:

**Anti-virus:** Go with Microsoft Security Essentials. That's all I used and I never had an issue. Lightweight and stayed out of my way.

**IDE:** I was ST3 all the way. I never got into using a "full" IDE like PHPStorm. Bonus because it was cross platform.

**Command Line:** a wing and a prayer. Normally PowerShell when I had to, but often I'd find small utilities that had a GUI just because it was easier and I'm lazy. I've seen some stuff recently that makes it better, but haven't dug into it.

**Vagrant / VVV:** I used MAMP for most of my time and only got Vagrant going shortly before I switched, and it was only on a single project. Eric Mann at 10up wrote a great thing on how to get it working: https://ttmm.io/tech/vagrant-windows/

**Performance Monitor:** I used the Windows Sysinternals Suite (https://technet.microsoft.com/en-us/sysinternals/bb545021.aspx) which had all the pieces I needed, and has a solid track record. Since you're building from scratch, you're not gonna have any bloatware to deal with, so you're already ahead of the game. Since you're using an SSD, you shouldn't have to do anything like defrag or whatnot.

**General Tips:** I would often keep my applications and constant use data (like a local dev environment) on the main SSD drive, but have a secondary internal drive (usually bigger, SATA, etc) for actual file storage, including things like DropBox which is _always_ writing and re-writing to the system.
